teuthology/report.py

```diff
@@ -513,10 +513,13 @@ def try_delete_jobs(run_name, job_ids, delete_empty_run=True):
     """
     log = init_logging()
 
-    if isinstance(job_ids, int):
-        job_ids = [str(job_ids)]
-    elif isinstance(job_ids, basestring):
-        job_ids = [job_ids]
+    if not isinstance(job_ids, list):
```

Comment:
I've chosen to not use 'str' or 'basestring' in case we have 'str' or 'unicode', the latter is missing in py3 too.

Reply:
okay. but in this case, what we are expecting is exactly "str" in py2 and in py3.

anyway, if you insist, i am fine with this as well.

Reply:
For python2 it can be `unicode`, since checking `bytes` is equivalent to `str` and checked above. For py3, it is `str`, since `bytes` is not an `str` any longer in py3.

Reply:
i don't think we can have unicode in py2 here.